docs_espressif/en/additionalfeatures/language-tools.rst

```diff
@@ -0,0 +1,145 @@
+Language Tools
+==============
+
+This module provides a language model tool for the ESP-IDF extension, allowing users to execute common ESP-IDF operations through VS Code's chat interface using the Language Model Tools API.
```

Comment:
IMO, this sounds too technical!

Title could be - Command Chat or ESP-IDF Chat Commands

Description:
This feature lets you run ESP-IDF commands directly from the VS Code chat window.
Instead of typing terminal commands, you can simply ask in chat - and the tool will execute common ESP-IDF actions for you, like building, flashing, or monitoring your project.